adoc/admin-updates.adoc

```diff
@@ -0,0 +1,20 @@
+=== Disable automatic updates
+
+Nodes added to a cluster have the service `skuba-update.timer` activated by default. This is the service responsible of running automatic updates. In case you require to completely disable automatic updates for a node it can be done by simply disabling the skuba-update service on the node:
+----
+systemctl disable --now skuba-update.timer
+----
+
```

Comment:
How do I know which node I'm disabling the update on? Would it maybe make sense to mention that you have to ssh into the node first?

Reply:
yes indeed. i will add something

Reply:
@nkoranova updated in the latest commit